packages/flutter_tools/lib/src/macos/macos_device.dart

```diff
@@ -96,9 +96,7 @@ class MacOSDevice extends Device {
       return LaunchResult.failed();
     }
 
-    // Make sure to call stop app after we've built.
```

Comment:
The Line 82 comment should be removed as well.

Reply:
Done